helpers/formathelpers.go

```diff
@@ -0,0 +1,66 @@
+package helpers
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/Appliscale/perun/intrinsicsolver"
+	"github.com/Appliscale/perun/logger"
+	"github.com/Appliscale/perun/offlinevalidator/template"
+	"github.com/awslabs/goformation"
+	"github.com/awslabs/goformation/cloudformation"
+	"github.com/ghodss/yaml"
+	"path"
+)
+
+func GetParser(filename string) (func([]byte, template.Template, *logger.Logger) (cloudformation.Template, error), error) {
+	templateFileExtension := path.Ext(filename)
+	if templateFileExtension == ".json" {
+		return ParseJSON, nil
+	} else if templateFileExtension == ".yaml" || templateFileExtension == ".yml" {
+		return ParseYAML, nil
+	} else {
+		return nil, errors.New("Invalid template file format.")
+	}
+}
+
+func ParseJSON(templateFile []byte, refTemplate template.Template, logger *logger.Logger) (template cloudformation.Template, err error) {
+
+	err = json.Unmarshal(templateFile, &refTemplate)
```

Comment:
Shouldn't there be line and column detection?

```
err = json.Unmarshal(templateFile, &refTemplate)
	if err != nil {
		if syntaxError, isSyntaxError := err.(*json.SyntaxError); isSyntaxError {
			syntaxOffset := int(syntaxError.Offset)
			line, character := lineAndCharacter(string(templateFile), syntaxOffset)
			logger.Error("Syntax error at line " + strconv.Itoa(line) + ", column " + strconv.Itoa(character))
		} else if typeError, isTypeError := err.(*json.UnmarshalTypeError); isTypeError {
			typeOffset := int(typeError.Offset)
			line, character := lineAndCharacter(string(templateFile), typeOffset)
			logger.Error("Type error at line " + strconv.Itoa(line) + ", column " + strconv.Itoa(character))
		}
		return template, err
	}
```

Reply:
The only thing I did was moving the function from one file to another :D 
I can certainly add this, seems like an important functionality.

Reply:
Oh, I see now that's a change from master, that wasn't included in the release_2_0 branch.